docs/conf.py

```diff
@@ -1,87 +1,90 @@
-import sys, os
+import os
+import sys
```

Comment:
**suggestion:** Since we're willing to open a PR on the [original repo](https://github.com/django-import-export/django-import-export), I believe we should respect their codestyle (even that their codestyle is different than ours).

If you proceed with this suggestion, I believe we should checks almost all files that were changed on this PR.